src/general/OptionsMenu.cs

```diff
@@ -370,12 +457,12 @@ private void SetSettingsTab(string tab)
     /// </summary>
     private float ConvertSoundBarToDb(float value)
     {
-        return (value - 100) / AUDIO_BAR_SCALE;
+        return GD.Linear2Db(value / 100.0f);
```

Comment:
Why was this change done? What does this do?

Reply:
It uses a more proper logarithm conversion to decibels, it also handles silencing the audio if it gets down to 0, vs it being at a really low volume like it was before. The mute checkbox still works on top of that. Unless I misunderstood something, that division by 6.0f ends up with notable amounts of error, like 10% volume should be -20 db but it's -15 with that.

Reply:
I suppose that's fine, then.